src/deploy/functions/deploy.ts

```diff
@@ -13,13 +12,13 @@ import * as gcfv2 from "../../gcp/cloudfunctionsv2";
 import * as utils from "../../utils";
 import * as backend from "./backend";
 
-const GCP_REGION = functionsUploadRegion;
+const DEFAULT_UPLOAD_REGION = process.env.FIREBASE_FUNCTIONS_UPLOAD_REGION;
```

Comment:
Do we actually need to have this env anymore?

Reply:
Thought the env var provided a config knobs to existing users so it is less breaking of a change.

Reply:
I'd start with the assumption that people can always upload to the region where they're deploying. I can't fathom a reason they'd need a different region.